google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/BlobSnippets.java

```diff
@@ -173,6 +173,22 @@ public Blob copyToStrings(String bucketName, String blobName) {
   }
 
   /**
+   * Example of moving a blob to a different bucket with a different name.
+   */
+  public Blob moveTo(Blob srcBlob, String destBucket, String destBlob) {
```

Comment:
Also - I wasn't sure if this sample was super relevant for the javadocs, since it's not about how to use either `copyTo` or `delete`, but how to do a move, which there isn't a method for..